com.unity.render-pipelines.universal/Runtime/Passes/DepthNormalOnlyPass.cs

```diff
@@ -14,9 +14,12 @@ public class DepthNormalOnlyPass : ScriptableRenderPass
         internal ShaderTagId shaderTagId { get; set; } = k_ShaderTagId;
 
         private RenderTargetHandle depthHandle { get; set; }
+        private RenderTargetHandle cameraDepthHandle { get; set; }
         private RenderTargetHandle normalHandle { get; set; }
         private FilteringSettings m_FilteringSettings;
 
+        private bool m_UseDepthPriming;
```

Comment:
This seems to be a pattern that has to be copied to several passes. The passes have access to the ScriptableRenderer, I wonder if we should expose a property on the renderer that tells if a depth prepass/priming is used.

Reply:
Fixed